app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from typing import Any
+from app.errors import (
```

Comment:
The import statement should use absolute imports instead of relative imports. Change `from app.errors import` to `from errors import` to comply with the checklist.